recipe/meta.yaml

```
@@ -34,18 +34,18 @@ requirements:
     - python
     - pip
     - setuptools
-    - cython >=0.29.33,<3
+    - cython >=0.29.33
     - numpy
```

Actually, I would add this pin for safety with the release of numpy 2 which will happen soon.
```suggestion
    - numpy <2
```

---

Thanks for reminding us ! Unfortunately I already uploaded to PyPI :( So I'm going to fix the pyproject and regenerate the wheels and make a new tag, then I'll update this PR to point to the new tag and fix the numpy pinning.

---

This is why I have the following comment above it:
https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy